internal/util/kubebuilder/project.go

```diff
@@ -20,12 +20,14 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const ConfigFile = "PROJECT"
+const (
+	configFile = "PROJECT"
```

Comment:
Used just here so should not be global.

Reply:
Seems reasonable, but nit on the terminology. This is still global. But now it is unexported.

Ditto re: changes in this file should be done in a separate PR.